src/lib/components/editor/AiTranslateToolbarButton.svelte

```diff
@@ -37,9 +39,18 @@
     let isErrorModalOpen = false;
     let isTranslating = false;
     let machineTranslations = machineTranslationStore.machineTranslations;
+    let isRetranslateModalOpen = false;
+    let retranslateReason = '';
+    let retranslateModelIsTransacting = false;
     $: machineTranslation = $machineTranslations.get(itemIndex);
     $: showTranslateButton = canShowAnything && !machineTranslation?.id;
     $: showRating = canShowAnything && !showTranslateButton && $userIsEqual(machineTranslation?.userId);
+    $: translatedLessThan24HoursAgo = resourceContent.machineTranslations.some(
```

Comment:
Should this variable name be 1 hour ago?

Reply:
Oh yeah, I thought the AC said 24 hours instead of 1.